bin/gltf-pipeline.js

```diff
@@ -49,7 +61,13 @@ var options = {
     binary : binary,
     embed : !separate,
     embedImage : !separateImage,
-    quantize : quantize
+    quantize : quantize,
+    ao_diffuse : ao_diffuse,
+    ao_scene : ao_scene,
+    ao_rayDepth : ao_rayDepth,
+    ao_resolution : ao_resolution,
+    ao_samples : ao_samples,
```

Comment:
Perhaps pack all the AO properties into an object. Also this makes it less verbose in `gltfPipeline`.